modules/base/rendering/renderabletrail.cpp

```diff
@@ -273,7 +273,7 @@ void RenderableTrail::render(const RenderData& data, RendererTasks&) {
     
     if (usingFramebufferRenderer) {
         glDepthMask(false);
-        glBlendFunc(GL_SRC_ALPHA, GL_ONE);
+        //glBlendFunc(GL_SRC_ALPHA, GL_ONE);
```

Comment:
We changed it to have better colors when generating the video for the SpaceX guys. The blending between the renderable trails and the atmosphere wasn't good. Should it be enabled again?